cmd/oceantv/broadcast.go

```diff
@@ -307,14 +308,67 @@ func extStart(ctx context.Context, cfg *BroadcastConfig, log func(string, ...int
 
 // extStop uses the OffActions in the provided broadcast config to perform
 // external streaming hardware shutdown.
+// If an OffAction is a controller action, we don't perform it yet.
 func extStop(ctx context.Context, cfg *BroadcastConfig, log func(string, ...interface{})) error {
 	if cfg.OffActions == "" {
 		return nil
 	}
 
-	err := setActionVars(ctx, cfg.SKey, cfg.OffActions, settingsStore, log)
-	if err != nil {
-		return fmt.Errorf("could not set device variables to end stream: %w", err)
+	vars := strings.Split(cfg.OffActions, ",")
+	if len(vars) == 0 {
+		return errors.New("no off actions to perform")
```

Comment:
Since we already check that OffActions are not nil :311, its not that there are no off actions, but there are no valid off actions.

Reply:
Actually, is it even possible for this to fail if the actions != ""?

Reply:
It's not possible, no. Will fix.